test/manifests/manifests.test.ts

```diff
@@ -92,56 +90,6 @@ describe('manifest ingest', () => {
     })
   })
 
-  test('updateManifestsFromRippled', async () => {
```

Comment:
This test case should be replaced with what it's doing now

Reply:
Yes, the new manifests retrieval is tested with the `update Manifests` test.